.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,18 +1,11 @@
 ### Description
-
 [Describe what this change achieves]
 
 ### Issues Resolved
-
 [List any issues this PR will resolve]
 
 ### Check List
-
-- [ ] New functionality includes testing.
-  - [ ] All tests pass
-- [ ] New functionality has been documented.
-- [ ] Commits are signed per the DCO using `--signoff`
```

Comment:
Did we want to remove all this?

Reply:
Following the org example pull request template here: https://github.com/opensearch-project/.github/blob/main/PULL_REQUEST_TEMPLATE.md